docs/articles/lambda_compilation.md

```diff
@@ -0,0 +1,33 @@
+﻿# Compilation of expressions to CLR lambdas
+
+If you need a better preformance you should use compilation of expressions to CLR lambdas feature. 
```

Comment:
Question about perf, it's still faster than Evaluate with the Compilation or it's recommended at the docs to say the consumer to cache the lambda?

Reply:
I can also later at MemoryCache package to add a option to cache the lambdas if the Compilation+MethodCall is slower than Evaluate, but this should be a separated PR I think

Reply:
Good idea, but we've already have an article about caching. Not sure about performance with a cached lambda